gcloud/translate/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Does this `__init__.py` need to be added to `IGNORED_MODULES`? https://github.com/GoogleCloudPlatform/gcloud-python/blob/master/scripts/verify_included_modules.py#L32